Proxy/internal/cluster/cluster.go

```diff
@@ -0,0 +1,48 @@
+package cluster
+
+import (
+	"context"
+	"os"
+	"time"
+
+	model "github.com/amosproj/amos2023ss04-kubernetes-inventory-taker/Proxy/internal/model"
+	"github.com/uptrace/bun"
+	"github.com/uptrace/bun/schema"
+	"gopkg.in/yaml.v2"
+	"k8s.io/klog"
+)
+
+type KubeConfig struct {
+	//nolint:tagliatelle
+	CurrentContext string `yaml:"current-context"`
```

Comment:
It's a field tag. It's used by the yaml package when it unmarshals the yaml data into the config struct of type KubeConfig. When it finds a "current-context" key in kubeconfig file, the value is assigned to the CurrentContext field. I can't change the formatting of the field tag because it is defined in the kubeconfig